src/web/Ligare/web/scaffolding/__main__.py

```diff
@@ -35,7 +35,7 @@ class ScaffoldParsedArgs(NamedTuple):
     name: Operation
     endpoints: list[Operation]
     template_type: Literal["basic", "openapi"]
-    modules: list[Literal["database", "test"]] | None
+    modules: list[str] | None
```

Comment:
Is this change directly related to the VScode module?

Reply:
It is more related to the ability to create more Scaffold modules without having to update the `modules` type or its values. We don't lose anything in practice; it has always been a list of strings, but now the specific values are unimportant to the type checker because they are dynamically set.